agent/sandbox/tests/test_novita.py

```
@@ -0,0 +1,280 @@
+        monkeypatch.setattr("agent.sandbox.providers.novita._get_novita_module", _raise)
+
+        with pytest.raises(SandboxProviderConfigError):
+            provider.initialize({"api_key": "sk_test"})
+
+
+class TestNovitaSandboxProviderExecution:
+    """Test NovitaSandboxProvider create/execute/destroy."""
+
+    def test_create_instance_returns_sandbox_id(self, monkeypatch):
+        sandbox = _FakeSandbox(sandbox_id="sbx-abc")
+        provider, fake_module = _build_provider(sandbox, monkeypatch)
```

Ruff (0.16.1)

[warning] Unpacked variable `fake_module` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)